model/build.gradle.kts

```diff
@@ -41,5 +42,9 @@ dependencies {
     implementation("com.sksamuel.hoplite:hoplite-core:$hopliteVersion")
     implementation("com.sksamuel.hoplite:hoplite-hocon:$hopliteVersion")
     implementation("com.vdurmont:semver4j:$semverVersion")
+    implementation("org.jetbrains.exposed:exposed-core:$exposedVersion")
```

Comment:
It is not so nice that the rather generic _model_ now depends on these modules, which represent a specific persistence technology. Would it be an option to introduce a new module ("database", "persistence", whatever) where such DB-related utility classes could live, so that they could be re-used by multiple modules?

Reply:
Sure, that could be an option. Another alternative would be to refactor the file archiver to not depend on the model anymore.
If you're ok with that I'd like to leave it for future improvement.

Reply:
I mostly agree with that, but the model module is not only a model anymore for some time. I think we should consider to either rename it to e.g. "core" or to move the logic classes into a separate module. I would suggest that we accept this for now and put this on the agenda for the dev meetings, as I believe it will require some discussion to find an agreement for the module structure.